pkg/config/config.go

```diff
@@ -170,9 +170,12 @@ func GetMemoryMax() uint64 {
 	// try to get the memory from the cgroup
 	memoryMax, err := getContainerMemory(memoryValueMax)
 	if err != nil {
-		log.Debugf("GetMemoryMax: Error while getting memory from cgroup: %v", err)
+		log.Warnf("GetMemoryMax: Error while getting memory from cgroup: %v", err)
 		// if we can't get the memory from the cgroup, get it from the OS
 		memoryMax = memory.TotalMemory()
+		log.Infof("GetMemoryMax: Memory from the OS in bytes: %v", memoryMax)
```

Comment:
nit: `...from the OS ...`  should be `...from the Host ..`

Reply:
also log the memoryMax in humanize.Comma format

Reply:
I changed it to log in MB, similar to how we do in other places.